app/views/security_check_failed/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<% self.title = t('security_check_failed.title') %>
```

Comment:
This looks like a good use-case for `StatusPageComponent`, which can handle most everything we're doing on this page? (icon, header, troubleshooting options).

Example: https://github.com/18F/identity-idp/blob/main/app/views/two_factor_authentication/_locked.html.erb